samples/peripheral/lpuart/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _lpuart_sample:
+
+Low Power UART sample
+#####################
+
+The Low Power UART sample demonstrates the capabilities of the :ref:`uart_nrf_lpuart` module.
+
+Overview
+********
+
+The sample implements a simple loopback using a single UART instance.
+The sample has console and logging disabled by default, to demonstrate low power consumption while having UART active.
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF52840DK|
```

Comment:
See this on how to add supported dev kits -> https://github.com/nrfconnect/sdk-nrf/pull/2632/files#diff-954121af6470d736bd2a2ef278913f88 (line 31)

Reply:
fixed